frontend/AppNavigator.js

```diff
@@ -1,41 +1,52 @@
 import { createStackNavigator } from "@react-navigation/stack";
-
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
 
 import HomePage from "./screens/homePage";
 import SubmitIpptPage from "./screens/ipptSubmissionPage";
 import LoginPage from "./screens/loginPage";
 import RegistrationPage from "./screens/registrationPage";
 import NsFitPage from "./screens/nsFitPage";
 
-
-const Drawer = createDrawerNavigator();
 const Stack = createStackNavigator();
+const Tab = createBottomTabNavigator();
 
-const HomeNavigator = () => {
+// BottomNavigator Bar
+const  BottomNavigator = () => {
     return(
-        <Stack.Navigator screenOptions={{ headerShown: false }}>
-            <Stack.Screen name="Login" component={LoginPage} />
-            <Stack.Screen name="Register" component={RegistrationPage} />
-            <Stack.Screen name="ViewIpptScore" component={HomePage} />
-            <Stack.Screen name="SubmitIppt" component={SubmitIpptPage} />
-            <Stack.Screen name="NsFit" component={NsFitPage} />
-        </Stack.Navigator>
-    )
+    <Tab.Navigator screenOptions={{ headerShown: false }} initialRouteName="View Ippt">
+        <Tab.Screen name="Submit Ippt" component={SubmitIpptPage}  options={{ tabBarIcon: () => (
+          <MaterialCommunityIcons 
+          name="file-upload" 
+          size={24} 
+          color="#a12427" /> )}
+        }/>
+        <Tab.Screen name="View Ippt" component={HomePage} options={{ tabBarIcon: () => (
+          <MaterialCommunityIcons 
+          name="home" 
+          size={24} 
+          color="#a12427" /> )}
+       }/>
+        <Tab.Screen name="NsFit" component={NsFitPage} options={{ tabBarIcon: () => (
+          <MaterialIcons
+          name="fitness-center" 
+          size={24} 
+          color="#a12427" /> )}
+        }/>
+    </Tab.Navigator>
+  );
 }
 
 export const AppNavigator = () => {
-    return(
-        <NavigationContainer>
-            <Drawer.Navigator initialRouteName="Login" screenOptions={{ headerShown: true }}>
-                <Drawer.Screen name="Login" component={LoginPage} />
-                <Drawer.Screen name="Register" component={RegistrationPage} />
-                <Drawer.Screen name="View Ippt Score" component={HomePage} options={{headerShown: false}}/>
-                <Drawer.Screen name="Submit Ippt" component={SubmitIpptPage} options={{headerShown: false}}/>
-                <Drawer.Screen name="Do Ns Fit" component={NsFitPage} options={{headerShown: false}}/>
-            </Drawer.Navigator>
-        </NavigationContainer>
+  return(
+    <NavigationContainer >
+      <Stack.Navigator>
+        <Stack.Screen name="Login" component={LoginPage} options={{headerShown: false}}/>
+        <Stack.Screen name="Register" component={RegistrationPage} options={{headerShown: false}}/>
+        <Stack.Screen name="HomeNavigator" component={BottomNavigator} options={{headerShown: false}} />
+      </Stack.Navigator>
+    </NavigationContainer>
```

Comment:
Good work at removing the login and registration page from the stack navigator as these pages shouldn't be accessed directly.